pkg/core/blockchain.go

```diff
@@ -29,7 +29,7 @@ import (
 // Tuning parameters.
 const (
 	headerBatchCount = 2000
-	version          = "0.0.9"
+	version          = "0.0.10"
```

Comment:
BTW, I think it makes sense to set it to 0.1.0 avoiding confusion with NEO 2.0 DBs (that should never leave 0.0.x series).